package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@viaprotocol/web3-wallets",
-  "version": "1.0.30",
+  "type": "module",
```

Comment:
Maybe I'm wrong, but it seems that this package can be used in another package only with the `"type": "module"`

I don't think it's necessary